tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/GitHubService.cs

```diff
@@ -311,5 +330,83 @@ public async Task<Issue> GetIssueAsync(string repoOwner, string repoName, int is
                 throw;
             }
         }
+
+        public async Task UpdateFileAsync(string owner, string repoName, string path, string message, string content, string sha, string branch)
+        {
+            try
+            {
+                var updateRequest = new UpdateFileRequest(message, content, sha, branch);
```

Comment:
It's better to add check that branch is not `main` to have be on the safer side.